kolibri/plugins/coach/assets/src/views/create-exam-page/index.vue

```diff
@@ -225,6 +257,11 @@
         }
         return '';
       },
+      generateRandomSeed() {
```

Comment:
@jonboiser I tend to lean towards solutions that don't require more dependencies. You have an opinion on this?